### PR TITLE
parser for openphish commercial feed

### PR DESCRIPTION
**Pull request by fpokorny**

### CODE REVIEW

**Comment by wagner-certat** [comment]
I also don't understand why this change is necessary or useful, please be more verbose!

https://www.openphish.com/feed.txt (see https://github.com/certtools/intelmq/blob/master/docs/Feeds.md#openphish) has one URL per line, no JSON there.

**Comment by fpokorny** [comment]
Ye sorry, it's my bad. The free openphish feed is just a textfile, however they offer free access for CSIRTs to their commercial feed which is a file with one json per line. So for the sake of intelmq I suppose it should be a "new" feed, something like openphish-commercial..

**Comment by wagner-certat** [comment]
Ok, so it's not a bugfix, but a new "feature"/feed.

Then please create a new entry in etc/feeds.yml describing the Feed, write the new parser+tests etc.

**Comment by fpokorny** [comment]
Okay, now it should be better. :-)

File: intelmq/etc/feeds.yaml [hunk 1]
```diff
@@ -105,6 +105,25 @@ providers:
...
+    Phishing:
+      description: OpenPhish is a fully automated self-contained platform for phishing
+        intelligence. It identifies phishing sites and performs intelligence analysis
+        in real time without human intervention and without using any external resources,
+        such as blacklists.
+      additional_information:
+      bots:
+        collector:
+          module: intelmq.bots.collectors.http.collector_http
+          parameters:
```
Comment (wagner-certat) [inline intelmq/etc/feeds.yaml]: The username/password parameters are missing here.

**Comment by fpokorny** [comment]
Should be fixed now.

**Comment by wagner-certat** [comment]
FYI, some small adaptions I made additionally: 91b0cca8

**Pull request merged**